src/web/router.js

```diff
@@ -111,11 +111,10 @@ router.get('/discrepancies/search', auth.secured(PermissionLevel.USER_SUPPORT),
 router.post('/discrepancies/search', auth.secured(PermissionLevel.USER_SUPPORT), discrepancies.getDiscrepancyReport)
 router.post('/discrepancies/resolve/:id', auth.secured(PermissionLevel.USER_SUPPORT), discrepancies.resolveDiscrepancy)
 
-router.get('/stripe/balance', stripe.balance)
-router.get('/stripe/basic/create', auth.secured(PermissionLevel.VIEW_ONLY), stripe.basic)
-router.post('/stripe/basic/create', auth.secured(PermissionLevel.USER_SUPPORT), stripe.basicCreate)
-router.get('/stripe/basic/create-test-account', auth.secured(PermissionLevel.VIEW_ONLY), stripe.createTestAccount)
-router.post('/stripe/basic/create-test-account', auth.secured(PermissionLevel.USER_SUPPORT), stripe.createTestAccountConfirm)
+router.get('/stripe/basic/create', auth.secured, stripe.basic)
```

Comment:
These have since changed on master (some work done by S.D), other than removing the new endpoint there shouldn't be any other changes here

Reply:
Urgh the conflict resolution didn't work

Reply:
Fixed